alienv

```diff
@@ -253,22 +257,46 @@ case "$ACTION" in
     MODULES=$(normModules "${ARGS[@]}")
     collectModules
     existModules $MODULES
-    eval $($MODULECMD bash add $MODULES 2> >(grep -v "$IGNORE_ERR" >&2))
+    PS1DEV=
+    if [[ $DEVOPT == 1 ]];then
+      PS1DEV=" (dev)"
+      for MODULE in "$MODULES"; do
+        # echo "source $WORK_DIR/$ARCHITECTURE/$MODULE/etc/profile.d/init.sh"
+        eval $($MODULECMD bash add $MODULES 2> >(grep -v "$IGNORE_ERR" >&2))
```

Comment:
I don't think it makes sense to *both* load the modulefile *and* source the `init.sh` script. They should declare the same variables anyway, and if they are incompatible, the conflicts would be a bit of a mess to sort out. Additionally, when `aliBuild` builds a package, it only sources the `init.sh` script, not the modulefile, so if you want to replicate the build environment as `aliBuild` would create it, then only source `init.sh`.

Reply:
I agree with this and i will remove it